src/ui/pages/settings/tabs/GeneralSettings.svelte

```diff
@@ -108,6 +109,35 @@
   {/snippet}
 </SettingItem>
 
+<SettingItem
+  name="Excluded Folders"
+  description="Folders to exclude from annotation collection (one per line)"
+  notices={[
+    {
+      type: "info",
+      text: "Files in these folders will not appear in the annotations view (e.g., <code>_templates/</code>)",
+    },
+    {
+      type: "warning",
+      text: "Changing this setting results in a re-indexation of the entire vault",
+    },
+  ]}
+>
+  {#snippet control()}
+    <TextArea
+      value={plugin.settings.excluded_folders.join("\n")}
+      placeholder="_templates/&#10;archive/"
+      onChange={(value) => {
```

Comment:
This should probably be debounced (could also be an onBlur callback, but that probably needs to be added in the components submodule).